sign/eddsa/eddsa_test.go

```diff
@@ -101,6 +101,35 @@ func TestEdDSASigning(t *testing.T) {
 	}
 }
 
+// Test signature malleability
+func TestEdDSASigningMalleability(t *testing.T) {
+	/* 2^252+27742317777372353535851937790883648493 */
+	var L []uint8 = []uint8{0xed, 0xd3, 0xf5, 0x5c, 0x1a, 0x63, 0x12, 0x58, 0xd6, 0x9c, 0xf7,
```

Comment:
as you're casting this to `uint16` anyway, might as well start with `uint16`

Reply:
OK, I'll change that.